fala/templates/adviser/results.html

```diff
@@ -0,0 +1,129 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <div class="govuk-grid-column-full">
+    <div id="google_translate_element" class="govuk-!-margin-bottom-6"></div>
+    <h1>Search Results</h1>
+    {% if data and data.count and data.count > 0 %}
+      {% if data.origin %}
+        {% trans page_count=data.results|length, result_count=data.count %}
+          {% pluralize %}
+          <span class="govuk-!-font-weight-bold">{{ result_count }} results</span> in order of closeness to
+        {% endtrans %}
+        <strong class="notranslate" translate="no">{{ form.postcode.value() }}</strong>
+        {%- if form.name.value() %}
+          {{ _('matching') }} <strong>{{ form.name.value() }}</strong>.
+        {%- else -%}
+          .
+        {% endif %}
+      {% else %}
+        {% trans result_count=data.count, page_count=data.results|length, org_name=form.name.value() %}
+          {% pluralize %}
+          <span class="govuk-!-font-weight-bold">{{ result_count }} results</span> for <strong>{{ org_name }}</strong>.
+        {% endtrans %}
+      {% endif %}
+      <ul class="govuk-list results-list">
+        {% for item in data.results %}
+          <li class="results-list-item">
+            {% if item.distance %}
+              <p class="miles-away govuk-body">
+                <span class="govuk-visually-hidden">{{ _('Distance') }}</span>
+                {% trans miles=item.distance|round(2) %}{{ miles }} miles away{% endtrans %}
+              </p>
+            {% endif %}
+            <h2 class="govuk-body-l govuk-!-font-weight-bold">{{ item.organisation.name }}</h2>
+            <div class="telephone">
```

Comment:
Sorry I found this CSS class isn't being used for anything (Not even in the stylesheets). So this can go bye bye!